Java 9 Support

### Operating System
Travis CI / Trusty
Ubuntu 14.04.5 LTS

### JDK 
Oracle JDK9
Java(TM) SE Runtime Environment (build 9.0.1+11)

### Expected Behavior
Codacy coverage reporter supports Java 9

### Actual Behavior

Codacy coverage reporter does crash with Java 9. 

```java
5:37:31,873 %PARSER_ERROR[coloredLevel] com.codacy Parsing coverage data... 
2018-03-16 15:37:32,166 %PARSER_ERROR[coloredLevel] com.codacy Uploading coverage data... 
Exception in thread "main" java.lang.NoClassDefFoundError: javax/activation/MimetypesFileTypeMap
	at rapture.mime.MimeTypes$.<init>(mime.scala:1544)
	at rapture.mime.MimeTypes$.<clinit>(mime.scala)
	at rapture.net.PostType$$anon$14.contentType(net.scala:132)
	at rapture.net.PostType$$anon$14.contentType(net.scala:131)
	at rapture.net.HttpSupport$$anon$1$$anonfun$doHttp$1.apply(http.scala:159)
	at rapture.net.HttpSupport$$anon$1$$anonfun$doHttp$1.apply(http.scala:131)
	at rapture.core.ThrowExceptionsMode.wrap(modes.scala:199)
	at rapture.net.HttpSupport$$anon$1.doHttp(http.scala:131)
	at rapture.net.HttpSupport$Capability$$anonfun$httpPost$1.apply(http.scala:78)
	at rapture.net.HttpSupport$Capability$$anonfun$httpPost$1.apply(http.scala:78)
	at rapture.core.ThrowExceptionsMode.wrap(modes.scala:199)
	at rapture.net.HttpSupport$Capability.httpPost(http.scala:78)
	at com.codacy.api.client.CodacyClient.post(CodacyClient.scala:56)
	at com.codacy.api.service.CoverageServices.sendReport(CoverageServices.scala:20)
	at com.codacy.CodacyCoverageReporter$$anonfun$coverageWithTokenAndCommit$1$$anonfun$apply$2$$anonfun$apply$3.apply(CodacyCoverageReporter.scala:150)
	at com.codacy.CodacyCoverageReporter$$anonfun$coverageWithTokenAndCommit$1$$anonfun$apply$2$$anonfun$apply$3.apply(CodacyCoverageReporter.scala:136)
	at com.codacy.CodacyCoverageReporter$.com$codacy$CodacyCoverageReporter$$transform(CodacyCoverageReporter.scala:186)
	at com.codacy.CodacyCoverageReporter$$anonfun$coverageWithTokenAndCommit$1$$anonfun$apply$2.apply(CodacyCoverageReporter.scala:135)
	at com.codacy.CodacyCoverageReporter$$anonfun$coverageWithTokenAndCommit$1$$anonfun$apply$2.apply(CodacyCoverageReporter.scala:135)
	at com.codacy.parsers.CoverageParserFactory$$anonfun$withCoverageReport$1.apply(CoverageParser.scala:42)
	at com.codacy.parsers.CoverageParserFactory$$anonfun$withCoverageReport$1.apply(CoverageParser.scala:40)
	at scala.Option.map(Option.scala:146)
	at com.codacy.parsers.CoverageParserFactory$.withCoverageReport(CoverageParser.scala:39)
	at com.codacy.CodacyCoverageReporter$$anonfun$coverageWithTokenAndCommit$1.apply(CodacyCoverageReporter.scala:135)
	at com.codacy.CodacyCoverageReporter$$anonfun$coverageWithTokenAndCommit$1.apply(CodacyCoverageReporter.scala:129)
	at com.codacy.api.helpers.FileHelper$$anonfun$withTokenAndCommit$1$$anonfun$apply$2.apply(FileHelper.scala:21)
	at com.codacy.api.helpers.FileHelper$$anonfun$withTokenAndCommit$1$$anonfun$apply$2.apply(FileHelper.scala:19)
	at scala.Option.map(Option.scala:146)
	at com.codacy.api.helpers.FileHelper$$anonfun$withTokenAndCommit$1.apply(FileHelper.scala:19)
	at com.codacy.api.helpers.FileHelper$$anonfun$withTokenAndCommit$1.apply(FileHelper.scala:18)
	at com.codacy.api.helpers.FileHelper$$anonfun$withCommit$2.apply(FileHelper.scala:34)
	at com.codacy.api.helpers.FileHelper$$anonfun$withCommit$2.apply(FileHelper.scala:33)
	at scala.Option.map(Option.scala:146)
	at com.codacy.api.helpers.FileHelper$.withCommit(FileHelper.scala:33)
	at com.codacy.api.helpers.FileHelper$.withTokenAndCommit(FileHelper.scala:18)
	at com.codacy.CodacyCoverageReporter$.coverageWithTokenAndCommit(CodacyCoverageReporter.scala:129)
	at com.codacy.CodacyCoverageReporter$.codacyCoverage(CodacyCoverageReporter.scala:162)
	at com.codacy.CodacyCoverageReporter$.main(CodacyCoverageReporter.scala:89)
	at com.codacy.CodacyCoverageReporter.main(CodacyCoverageReporter.scala)
Caused by: java.lang.ClassNotFoundException: javax.activation.MimetypesFileTypeMap
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:185)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
	... 39 more
```


### Reproducible Test Case

travis.yml
```yaml
dist: trusty
language: java
jdk:
  - oraclejdk9

# Upload test coverage to codacy
env:
  global:
    CODACY_PROJECT_TOKEN=[your-token]

before_install:
  - sudo apt-get install jq
  - wget -O ~/codacy-coverage-reporter-assembly-latest.jar $(curl https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/latest | jq -r .assets[0].browser_download_url)

after_success:
  - java -jar ~/codacy-coverage-reporter-assembly-latest.jar -l Java -r build/reports/jacoco/jacocoRootReport/jacocoRootReport.xml
```

--- a/README.md
+++ b/README.md
@@ -1,10 +1,10 @@
 # Codacy Coverage Reporter
 
-Multi-language coverage reporter for Codacy https://www.codacy.com
-
 [![Codacy Badge](https://api.codacy.com/project/badge/grade/1c524e61cd8640e79b80d406eda8754b)](https://www.codacy.com/app/Codacy/codacy-coverage-reporter)
 [![Build Status](https://circleci.com/gh/codacy/codacy-coverage-reporter.png?style=shield&circle-token=:circle-token)](https://circleci.com/gh/codacy/codacy-coverage-reporter)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.codacy/codacy-coverage-reporter/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.codacy/codacy-coverage-reporter)
+
+Multi-language coverage reporter for Codacy https://www.codacy.com
 
 ## Requirements
 

--- a/build.sbt
+++ b/build.sbt
@@ -27,7 +27,8 @@ libraryDependencies ++= Seq(
   caseApp,
   raptureJsonPlay,
   scalaTest,
-  cats
+  cats,
+  javaxActivation
 )
 
 mainClass in assembly := Some("com.codacy.CodacyCoverageReporter")

--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -2,15 +2,17 @@ import sbt._
 
 object Dependencies {
 
-  lazy val codacyScalaApi = "com.codacy" %% "codacy-api-scala" % "3.0.7"
-  lazy val coverageParser = "com.codacy" %% "coverage-parser" % "2.0.9"
-  lazy val caseApp = "com.github.alexarchambault" %% "case-app" % "1.2.0"
-  lazy val raptureJsonPlay = "com.propensive" %% "rapture-json-play" % "2.0.0-M8"
+  val codacyScalaApi = "com.codacy" %% "codacy-api-scala" % "3.0.7"
+  val coverageParser = "com.codacy" %% "coverage-parser" % "2.0.9"
+  val caseApp = "com.github.alexarchambault" %% "case-app" % "1.2.0"
+  val raptureJsonPlay = "com.propensive" %% "rapture-json-play" % "2.0.0-M8"
+  val javaxActivation = "com.sun.activation" % "javax.activation" % "1.2.0"
 
-  lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.0.5" % "test"
+  val scalaTest = "org.scalatest" %% "scalatest" % "3.0.5" % "test"
 
-  lazy val cats = "org.typelevel" %% "cats-core" % "1.0.1"
+  val cats = "org.typelevel" %% "cats-core" % "1.0.1"
 
-  lazy val logback = "ch.qos.logback" % "logback-classic" % "1.2.3"
-  lazy val log4s = "org.log4s" %% "log4s" % "1.5.0"
+  val logback = "ch.qos.logback" % "logback-classic" % "1.2.3"
+  val log4s = "org.log4s" %% "log4s" % "1.5.0"
+
 }

